src/worker/builder.rs

```diff
@@ -18,6 +31,10 @@ pub struct WorkerBuilder<E> {
     callbacks: CallbacksRegistry<E>,
     shutdown_timeout: Option<Duration>,
     shutdown_signal: Option<ShutdownSignal>,
+    tls_kind: TlsKind,
+    #[cfg(any(feature = "native_tls", feature = "rustls"))]
+    #[cfg_attr(docsrs, doc(cfg(any(feature = "native_tls", feature = "rustls"))))]
+    verify_server_cert: bool,
```

Comment:
I think I don't love us making `verify_server_cert` such a simple thing to turn off. I think I'd actually prefer that we make it very easy to connect with TLS normally (which you've already done with this PR), and then to say that any TLS customization requires going via `Worker::connect_with` and providing your own TLS connection.

Reply:
Hmm, I've used the [`imap`](https://github.com/jonhoo/rust-imap/blob/0f57af374669148a04a49f9151f83d4f5cb2ef34/src/client_builder.rs#L227-L229) crate for reference. I also think we are giving warnings that are standard, like `danger` prefixing and we are not encouraging people to turn it off.

Removing?

Reply:
I'm actually wondering whether it was a mistake for `imap` too :sweat_smile: For testing, one can simply run without TLS enabled (rather than using a self-signed cert). Hmm hmm hmm. I _think_ I'd prefer to leave this functionality out, at least for now, and then we can consider adding it if we get requests down the line.

Reply:
dropped it and the related code